src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -38,6 +40,111 @@ def load_profile_data():
         return json.load(f)
 
 
+def print_benchmark_summary(results):
+    output = "\n[ Benchmark Summary ]"
+    for prompt_size, avg_first_token_latency, avg_token_to_token_latency in results:
+        output += (
+            f"\n  Prompt size: {prompt_size}, "
+            f"Average first-token latency: {avg_first_token_latency:.4f} sec"
+        )
+        output += (
+            f", Average token-token latency: {avg_token_to_token_latency:.4f} sec"
```

Comment:
Sorry for finding this in the second review, but would you be averse to changing this to "token-to-token latency"? To be consistent with the graph and README verbiage. Looks good to me otherwise!

Reply:
Thanks for spotting. Changed to "token-to-token"